app/src/models/transfer.ts

```diff
@@ -33,6 +33,8 @@ export interface StoredTransfer extends RawTransfer {
   sendResult?: toEthereum.SendResult | toPolkadot.SendResult
   // A subscan unique Id shared accross chains to track ongoing transfers
   uniqueTrackingId?: string
+  // Transfer state
+  state?: string
```

Comment:
It should be 'status' 👍 
```suggestion
  status?: string
```